### PR TITLE
Pr fixes

### PR DESCRIPTION
**Pull request by RogPodge**
## Overview
Addresses comments left on
https://github.com/microsoft/MixedRealityToolkit-Unity/pull/10568
https://github.com/microsoft/MixedRealityToolkit-Unity/pull/10574
https://github.com/microsoft/MixedRealityToolkit-Unity/pull/10561

### CODE REVIEW

**Review by davidkline-ms — APPROVED** [review]

File: Assets/MRTK/Providers/Oculus/XRSDK/OculusXRSDKDeviceManager.cs [hunk 1]
```diff
@@ -48,8 +48,11 @@ public class OculusXRSDKDeviceManager : XRSDKDeviceManager
         public override void Initialize()
         {
             base.Initialize();
+#if !UNITY_2020_1_OR_NEWER
             UnityEngine.Debug.Log(@"Detected a potential deployment issue for the Oculus Quest. In order to use hand tracking with the Oculus Quest, download the Oculus Integration Package from the Unity Asset Store and run the Integration tool before deploying.
 The tool can be found under <i>Mixed Reality > Toolkit > Utilities > Oculus > Integrate Oculus Integration Unity Modules</i>");
+            // Add a note about abandoning Oculus XRSDK if on 2020+, use open xr instead please
```
Comment (keveleigh) [inline Assets/MRTK/Providers/Oculus/XRSDK/OculusXRSDKDeviceManager.cs:54]: Is this a TODO, or is this the note?

File: Assets/MRTK/Providers/Oculus/XRSDK/OculusXRSDKDeviceManagerProfile.cs [hunk 2]
```diff
@@ -117,33 +117,18 @@ public OVRHand.TrackingConfidence MinimumHandConfidence
...
         [Range(0, 5)]
-        private int defaultCpuLevel = 2;
+        private OVRManager.ProcessorPerformanceLevel defaultCpuLevel = OVRManager.ProcessorPerformanceLevel.SustainedHigh;
 
         /// <summary>
         /// Accessor for the Oculus CPU performance level.
         /// https://developer.oculus.com/documentation/native/android/mobile-power-overview
         /// </summary>
-        public int CPULevel
+        public OVRManager.ProcessorPerformanceLevel CPULevel
```
Comment (keveleigh) [inline Assets/MRTK/Providers/Oculus/XRSDK/OculusXRSDKDeviceManagerProfile.cs:131]: Also, were these changes intended for this PR? I don't see a reference to them in the description

File: Assets/MRTK/Providers/Oculus/XRSDK/OculusXRSDKDeviceManagerProfile.cs [hunk 3]
```diff
@@ -189,8 +190,8 @@ public void ApplyConfiguredPerformanceSettings()
         {
 #if OCULUSINTEGRATION_PRESENT
             UnityEngine.XR.XRSettings.eyeTextureResolutionScale = resolutionScale;
-            OVRManager.cpuLevel = CPULevel;
-            OVRManager.gpuLevel = GPULevel;
+            OVRManager.suggestedCpuPerfLevel = CPULevel;
```
Comment (keveleigh) [inline Assets/MRTK/Providers/Oculus/XRSDK/OculusXRSDKDeviceManagerProfile.cs:193]: Is this a change specific to a certain Oculus package? Or is it backwards compatible?

File: Assets/MRTK/Providers/Oculus/XRSDK/MRTK-Quest/Editor/OculusXRSDKHandtrackingConfigurationChecker.cs [hunk 4]
```diff
@@ -228,11 +228,9 @@ static void UpdateCSC()
             List<string> warningNumbers = new List<string>();
 
             // List of new warning numbers to add to the csc file
+            // Empty as of Oculus Integration 39.0 https://developer.oculus.com/downloads/package/unity-integration/39.0
```
Comment (keveleigh) [inline Assets/MRTK/Providers/Oculus/XRSDK/MRTK-Quest/Editor/OculusXRSDKHandtrackingConfigurationChecker.cs:231]: Interesting...do we need this List and any code path that uses it at all then? As a follow-up, do we need to stay compatible with previous versions of the Integration Package?

File: Assets/MRTK/Providers/Oculus/XRSDK/OculusXRSDKDeviceManager.cs (hunk 1 above)
Comment (RogPodge) [reply Assets/MRTK/Providers/Oculus/XRSDK/OculusXRSDKDeviceManager.cs:54]: It's a stray comment that's no longer needed. Removing in a followup

File: Assets/MRTK/Providers/Oculus/XRSDK/OculusXRSDKDeviceManagerProfile.cs (hunk 2 above)
Comment (RogPodge) [reply Assets/MRTK/Providers/Oculus/XRSDK/OculusXRSDKDeviceManagerProfile.cs:131]: Snuck these into this PR by accident, these are for compatibility with the latest Oculus package.

File: Assets/MRTK/Providers/Oculus/XRSDK/MRTK-Quest/Editor/OculusXRSDKHandtrackingConfigurationChecker.cs (hunk 4 above)
Comment (RogPodge) [reply Assets/MRTK/Providers/Oculus/XRSDK/MRTK-Quest/Editor/OculusXRSDKHandtrackingConfigurationChecker.cs:231]: The code path is in the Oculus Integration Package itself, so not much we can do there.

Oculus doesn't really highlight each package release, so I don't think backwards compatibility should be the highest priority considering their release patterns.
